### ISSUE TITLE
"Cast to [string] failed for value..." Error

### ISSUE THREAD

**Issue by Gdark42** [gab-ai-inc/gab-dissenter-extension#84]
Trying to Dissent to this specific url: https://i.imgur.com/07iG4J5.jpg

Using Brave browser with manually installed Dissenter v0.18, got this error:

Cast to [string] failed for value "[{"url":"https://i.imgur.com/07iG4J5.jpg","headers":{"User-Agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2227.0 Safari/537.36","Host":"i.imgur.com","Accept-Encoding":"application/gzip"},"timeout":10000,"gzip":true,"encoding":null}]" at path "pagePreview.images"

Manually installed Dissenter v0.19 on Brave, I still get this same error at that specific url.

Went to https://dissenter.com/ in web browser, pasted in that specific url to "create discussion", still get that same error.

**Comment by justagoodperson** [comment]
Yup, I can confirm that I receive the same error. I also got the same error trying to load Dissenter on a different image file. It's definitely something that's blowing up on the backend as opposed to with the browser extension, seems to me that it builds a preview for the Discussion page but that it's not setup to handle standalone images properly, some backend dev will have to take a look at that.
